raiden/tests/property/smart_contracts/test_nettingchannel.py

```diff
@@ -0,0 +1,333 @@
+# -*- coding: utf-8 -*-
+from ethereum.tester import TransactionFailed
+from hypothesis.stateful import GenericStateMachine
+from hypothesis.strategies import (
+    integers,
+    just,
+    one_of,
+    sampled_from,
+    tuples,
+)
+
+from raiden.tests.utils.tester import (
+    deploy_channelmanager_library,
+    deploy_nettingchannel_library,
+    new_channelmanager,
+    new_nettingcontract,
+    new_registry,
+    new_token,
+)
+from raiden.tests.property.smart_contracts.strategies import (
+    direct_transfer,
+    signed_transfer,
+)
+from raiden.tests.fixtures.tester import tester_state
+from raiden.utils import privatekey_to_address, sha3
+
+DEPOSIT = 'deposit'
+CLOSE = 'close'
+UPDATE_TRANSFER = 'updateTransfer'
+MINE = 'mine'
+
+
+class NettingChannelStateMachine(GenericStateMachine):
```

Comment:
Maybe add a small docstring here explaining what this class does and linking to the [stateful testing](http://hypothesis.readthedocs.io/en/master/stateful.html?highlight=execute_step#generic-state-machines) docs of Hypothesis, so that people unfamiliar with the library can read up on it.